scripts/dataset/README.md

```diff
@@ -9,7 +9,7 @@ You can change the configuration in the appropriate config file in the `config`
 ```json
 {
   "dataset": {
-    "declarationsRepositoryName": "Name of the services declarations repository",
+    "title": "Name of the services declarations repository",
```

Comment:
It's not a good thing that we duplicate the documentation between this README and the main one, this will lead to documentation rot and it has already diverged. Let's remove one.